### ISSUE TITLE
Accessibility: Inspect: The LegacyIAccessible.DefaultAction property should not be empty for Collapsed/Expanded DateTimePicker

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#6317]
* .NET Core Version: 
.Net 7.0 SDK build: 7.0.100-alpha.1.21568.2

* Have you experienced this same bug with .NET Framework?: 
Yes

**Minimal repro:**
1. Create a Winforms .Net Core application
2. Add a DateTimePicker control to form designer
3. Build and run application, then use Inspect to detect it when DateTimePicker is collapsed and expanded, pay attention to the LegacyIAccessible.DefaultAction property

**Actual behavior:**
LegacyIAccessible.DefaultAction property is empty when DateTimePicker is collapsed or expanded
Collapse:
![image](https://user-images.githubusercontent.com/26474449/145175364-c96c55d5-d7ec-4f48-9f89-61ba9f3900b2.png)
Expand:
![image](https://user-images.githubusercontent.com/26474449/145175531-4ee7aa87-f838-4270-bade-15cc457c3934.png)
**Expected behavior:**
LegacyIAccessible.DefaultAction property should be same as ComboBox control when it is collapsed or expanded
ComboBox is collapsed:
![image](https://user-images.githubusercontent.com/26474449/145176345-f3341db1-b869-42c4-bc3b-4b906dc75173.png)
ComboBox is expanded:
![image](https://user-images.githubusercontent.com/26474449/145176483-798c7445-b5b8-4292-91b7-b09ea296c9ab.png)

**Comment by MelonWang1** [comment]
Verified with .NET SDK 7.0.100-preview.3.22153.11 build, this issue is fixed.
DateTimePicker expanded:
![expand](https://user-images.githubusercontent.com/94418985/156720382-ae0fa505-5e07-45c5-838e-ad4942027311.png)
DateTimePicker collapsed:
![collapsed](https://user-images.githubusercontent.com/94418985/156720422-6d31163d-1d9a-49c2-8342-1c0ea1fd67ba.png)
